projects/plugins/jetpack/extensions/blocks/story/story.php

```diff
@@ -215,6 +215,7 @@ class="wp-block-jetpack-story_image wp-story-image %3$s"
 		);
 	}
 
+	// @phan-suppress-next-line PhanPossiblyUndeclaredVariable -- $width and $height are set when $image is truthy, and other scenarios return early.
```

Comment:
Any particular reason not to just set `$src`, `$width`, and `$height` (to `''`, `null`, and `null`) at the top of the function?